xivo_confgen/plugins/pjsip_conf.py

```diff
@@ -276,6 +276,9 @@ def _get_user_endpoint(self, user_sip, pickup_groups):
         for key, value in all_options:
             user_dict[key] = value
 
+        if user_sip.application_uuid:
+            user_dict['context'] = 'wazo-app-{}'.format(user_sip.application_uuid)
```

Comment:
I think this should be `stasis-wazo-app-{}`
https://blogs.asterisk.org/2019/03/27/stasis-improvements-goodbye-dialplan/

Reply:
Yes you can make a PR if you want. Don't forget to change wazo-calld too
https://github.com/wazo-pbx/wazo-calld/blob/master/wazo_calld/plugins/applications/stasis.py#L20

Reply:
Did you test it?

Reply:
No I was just trying to remember the prefix that I needed to use for my calld test and noticed that there was no prefix here.

I used `dialplan show stasis-<TAB>`  to confirm the auto-generated context name

Reply:
fixed